Update faq.md (#4607)
segmentio/segment-docs: src/connections/storage/warehouses/faq.md

Data in your warehouse is formatted into **schemas**, which involve a detailed description of database elements (tables, views, indexes, synonyms, etc.)
and the relationships that exist between elements. Segment's schemas use the following template: <br/>`<source>.<collection>.<property>`, for example,
`segment_engineering.tracks.user_id`, where source refers to the source or project name (segment_engineering), collection refers to the event (tracks),

> and the property refers to the data being collected (user_id). Note that it is not possible to have different sources feed data into the same schema in your warehouse. In fact, you will not have an option to use a schema name that is already associated with another source while setting up a new one. 

Reviewer: ```suggestion
and the property refers to the data being collected (user_id). **Note:** It is not possible to have different sources feed data into the same schema in your warehouse. While setting up a new schema, you cannot use a duplicate schema name. 
```
